docs/data/install-hadoop-2-0-0.md

```diff
@@ -0,0 +1,1499 @@
+
+The `user-vo-map` file is a simple, space-separated format that contains 2 columns; the first is a unix username and the second is the VO which that username correspond to. In order to create it you need to configure the gums client.
+
+The primary configuration file for the gums-client utilities is located in `/etc/gums/gums-client.properties`. The two properties that you must change are:
+
+|               |               |                                                                                                                                                                                         |
+|---------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Attribute     | Needs Editing | Value                                                                                                                                                                                   |
+| gums.location | Yes           | This should be set to the admin URL for your gums server, usually of the form gums.location=<https://GUMS_HOSTNAME:8443/gums/services/GUMSAdmin>                                        |
+| gums.authz    | Yes           | This should be set to the authorization interface URL for your gums server, usually of the form gums.authz=<https://GUMS_HOSTNAME:8443/gums/services/GUMSXACMLAuthorizationServicePort> |
+
+After the gums client is configured to generate the file run the following once by hand:
+
+``` console
+root# gums-host-cron
+```
+
+`user-vo-map` should be created in the following location:
+
+    /var/lib/osg/user-vo-map
+
+To have cron regularly update this file start the following service:
+
+``` console
+root# service gums-client-cron start
+```
+
+Make sure the **UserVOMapFile** field is set to this location in
+
+    /etc/gratia/gridftp-transfer/ProbeConfig
+
+Without `user-vo-map` , all gridftp transfers will show up as belonging to the VO "Unknown".
+
+### Using Gridmap based authorization mode
+
+Note: If you are using this mode for authorization, make sure the files /etc/grid-security/gsi-authz.conf and /etc/grid-security/prima-authz.conf do not exist.
+
+In order to enable generation of grid-mapfile and osg-user-vo-map.txt by using the edg-mkgridmap cron process to get information form VOMS servers do the following:
+
+``` console
+edg-mkgridmap 
+```
+
+If you have not installed this package, you will need to run `yum install edg-mkgridmap` first.
+
+
+### Validation
+
+Run the Gratia probe once by hand to check for functionality:
+
+``` console
+root# /usr/share/gratia/gridftp-transfer/GridftpTransferProbeDriver
+```
+
+Look for any abnormal termination and report it if it is a non-trivial site issue. Look in the log files in `/var/log/gratia/<date>.log` and make sure there are no error messages printed.
+
+BeStMan Configuration
+---------------------
```

Comment:
The contents of this section (except for the "BeStManHadoop-specific configuration" section and the first line of the "Copy certificates to bestman location " section) can be entirely replaced by a link to https://opensciencegrid.github.io/docs/data/bestman-install